content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,52 @@
+
+# How  to publish your recognizer
+We recommend using our [Recognizer Template in GitHub](https://github.com/ecoacoustics/recognizer-template) for publishing your recognizer. If you are familiar with GitHub you can proceed directly to the template. However, if you need more information on how to use it effectively, please continue reading this article.
+
+## What is GitHub and why is it great for this purpose?
+GitHub is a web-based service that utilises Git, which is a tool for version control. Git allows you to track changes, collaborate with others and manage different versions of your work. GitHub provides a platform for hosting and sharing Git repositories. You can read more about git and its uses from [GitHub documentation page](https://docs.github.com/en/get-started/using-git/about-git).
+
+### Version Control
+Version control is a system for recording changes to files over time, allowing you to track any modifications and revert back to previous versions. Version control is great for making collaboration with others seamless and manageable.
+
+### Zenodo - DOI generation
+Zenodo is an open-source project for generating DOIs that integrates well with GitHub. We recommend using Zenodo for generating a permanent DOI (Digital Object Identifier) for your recognizer. A DOI provides a unique and persistent identifier for your work, enabling others to cite it in their publications. Obtaining a DOI through Zenodo enhances the discoverability and visibility of yopur recognizer contributing to the impact and recoginition of your work within the scientific community.
+
+## How do I use the GitHub template to publish my recognizer?
+
+### Step 1.  Create a GitHub account 
+If you don't have a GitHub account yet,  the first step is to create one. You can create an account in [GitHub sign-in page](https://github.com/signup?ref_cta=Sign+up) following the prompts to set up your profile.
+
+### Step 2. Use the template as a start for a repository 
+Once you have a GitHub account, go to the [Recognizer Template repository](https://github.com/ecoacoustics/recognizer-template). You can use the template as a base for creating a new repository by clicking the green button that says "Use this template" and selecting "Create a new repository".  
+![The Use this template -button is located on the top right side of the page.](github-use-template.png) In the following page, input a name for your recognizer. You can choose if you want your repository to be public or private. Leave the the box that says "include all branches" unticked. Click "Create repository from template". For more detailed instructions please refer to the [README file](https://github.com/ecoacoustics/recognizer-template#readme) in the template.
```

Comment:
```suggestion
![The Use this template button is located on the top right side of the page.](github-use-template.png)

Then, in the following page:

- input a name for your recognizer repository. 
 
{{% hint info%}}
You should use `kebab-case` (all lower case, using hyphens instead of spaces) to title your repository as this is the convention used for most repositories on GitHub. For example:

- A repository that contained a recognizer for a _Red-tailed Black-cockatoo_ might be named `red-tailed-black-cockatoo-recognizer`.
- a project that detects frogs in SE Queensland might be named `se-queensland-frogs-recognizer`
- repository that hosts all the recognizers for your group might simply be called `ecoacoustics-recognizers`

{{% /hint %}}
   
 - You can choose if you want your repository to be public or private. Private is a good choice if you're not ready to publically publish but you want to get started. See below <insert link> for more information.
 
- Leave the box that says _include all branches_ unticked.
- Click _Create repository from template_. 
- Then review the instructions in the `README.md` file in your new repository
```